services/docker-stack-li-ps.yml

```diff
@@ -96,49 +66,7 @@ services:
     deploy:
       placement:
         constraints:
-          # - node.hostname != CA-RaTim-CO-IOCSrv
-          # - node.hostname != IA-14RaDiag03-CO-IOCSrv
-          # - node.hostname != IA-16RaBbB-CO-IOCSrv
-          # - node.hostname != IA-18RaDiag04-CO-IOCSrv
-          # - node.hostname != IA-20RaDiag01-CO-IOCSrv-1
-          - node.hostname != IA-20RaDiag01-CO-IOCSrv-2
-          # - node.hostname != IA-20RaDiag02-CO-IOCSrv-1
-          # - node.hostname != IA-20RaDiag02-CO-IOCSrv-2
-          - node.hostname != CA-RaCtrl-CO-Srv-1
-          - node.hostname != LA-RaCtrl-CO-Srv-1
-          - node.hostname != TA-TiRack-CO-FWSrv-1
-      replicas: 1
-      restart_policy:
-        condition: any
-    logging:
-      driver: "json-file"
-      options:
-        max-file: "10"
-        max-size: "10m"
-    networks:
-      - ioc-network
-
-  diag:
-    image: dockerregistry.lnls-sirius.com.br/fac/fac-iocs:__FAC_IOC_TAG_TEMPLATE__
-    depends_on:
-      - "spect-quads-lens"
-      - "corrs"
-      - "slnds"
-    command: bash -c 'source /etc/profile && /ioc-apps/li-ps-diag.bash'
-    deploy:
-      placement:
-        constraints:
-          # - node.hostname != CA-RaTim-CO-IOCSrv
-          # - node.hostname != IA-14RaDiag03-CO-IOCSrv
-          # - node.hostname != IA-16RaBbB-CO-IOCSrv
-          # - node.hostname != IA-18RaDiag04-CO-IOCSrv
-          # - node.hostname != IA-20RaDiag01-CO-IOCSrv-1
-          - node.hostname != IA-20RaDiag01-CO-IOCSrv-2
-          # - node.hostname != IA-20RaDiag02-CO-IOCSrv-1
-          # - node.hostname != IA-20RaDiag02-CO-IOCSrv-2
-          - node.hostname != CA-RaCtrl-CO-Srv-1
-          - node.hostname != LA-RaCtrl-CO-Srv-1
-          - node.hostname != TA-TiRack-CO-FWSrv-1
```

Comment:
is this correct?

Reply:
no, it is not. in my local repo your excerpt corresponds to the old version. and I have no local change to commit or local commit to push up to origin. Did you paste the old version? i do not understand...

Reply:
what I pointed was that the `diag` block was removed instead of updated. Is this correct?

Reply:
I see... this diag IOC is missing from the new config file, right? I might have have forgotten to add it as a service. i will check here...

Reply:
committed missing li-ps-diag service!